readme.md

```diff
@@ -0,0 +1 @@
+# fix linters errors
```

Comment:
- [ ] Branch should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), kindly consider using this in future projects (you can skip it for this one to avoid deleting the branch)